src/app/Task10/task10.js

```diff
@@ -0,0 +1,358 @@
+  },
+  {
+    name: "Lievijne Jans",
+    sex: "f",
+    born: 1542,
+    died: 1582,
+    father: null,
+    mother: null,
+  },
+  {
+    name: "Bernardus de Causmaecker",
+    sex: "m",
+    born: 1721,
+    died: 1789,
+    father: "Lieven de Causmaecker",
+    mother: "Livina Haverbeke",
+  },
+  {
+    name: "Jacoba Lammens",
+    sex: "f",
+    born: 1699,
+    died: 1740,
+    father: "Lieven Lammens",
+    mother: "Livina de Vrieze",
+  },
+  {
+    name: "Pieter de Decker",
+    sex: "m",
+    born: 1705,
+    died: 1780,
+    father: "Joos de Decker",
+    mother: "Petronella van de Steene",
+  },
+  {
+    name: "Joanna de Pape",
+    sex: "f",
+    born: 1654,
+    died: 1723,
+    father: "Vincent de Pape",
+    mother: "Petronella Wauters",
+  },
+  {
+    name: "Daniel Haverbeke",
+    sex: "m",
+    born: 1652,
+    died: 1723,
+    father: "Lieven Haverbeke",
+    mother: "Elisabeth Hercke",
+  },
+  {
+    name: "Lieven Haverbeke",
+    sex: "m",
+    born: 1631,
+    died: 1676,
+    father: "Pieter Haverbeke",
+    mother: "Anna van Hecke",
+  },
+  {
+    name: "Martina de Pape",
+    sex: "f",
+    born: 1666,
+    died: 1727,
+    father: "Vincent de Pape",
+    mother: "Petronella Wauters",
+  },
+  {
+    name: "Jan Francies Haverbeke",
+    sex: "m",
+    born: 1725,
+    died: 1779,
+    father: "Pieter Bernard Haverbeke",
+    mother: "Livina de Vrieze",
+  },
+  {
+    name: "Maria Haverbeke",
+    sex: "m",
+    born: 1905,
+    died: 1997,
+    father: "Emile Haverbeke",
+    mother: "Emma de Milliano",
+  },
+  {
+    name: "Petronella de Decker",
+    sex: "f",
+    born: 1731,
+    died: 1781,
+    father: "Pieter de Decker",
+    mother: "Livina Haverbeke",
+  },
+  {
+    name: "Livina Sierens",
+    sex: "f",
+    born: 1761,
+    died: 1826,
+    father: "Jan Sierens",
+    mother: "Maria van Waes",
+  },
+  {
+    name: "Laurentia Haverbeke",
+    sex: "f",
+    born: 1710,
+    died: 1786,
+    father: "Jan Haverbeke",
+    mother: "Maria de Rycke",
+  },
+  {
+    name: "Carel Haverbeke",
+    sex: "m",
+    born: 1796,
+    died: 1837,
+    father: "Pieter Antone Haverbeke",
+    mother: "Livina Sierens",
+  },
+  {
+    name: "Elisabeth Hercke",
+    sex: "f",
+    born: 1632,
+    died: 1674,
+    father: "Willem Hercke",
+    mother: "Margriet de Brabander",
+  },
+  {
+    name: "Jan Haverbeke",
+    sex: "m",
+    born: 1671,
+    died: 1731,
+    father: "Lieven Haverbeke",
+    mother: "Elisabeth Hercke",
+  },
+  {
+    name: "Anna van Hecke",
+    sex: "f",
+    born: 1607,
+    died: 1670,
+    father: "Paschasius van Hecke",
+    mother: "Martijntken Beelaert",
+  },
+  {
+    name: "Maria Sturm",
+    sex: "f",
+    born: 1835,
+    died: 1917,
+    father: "Charles Sturm",
+    mother: "Seraphina Spelier",
+  },
+  {
+    name: "Jacobus Bernardus van Brussel",
+    sex: "m",
+    born: 1736,
+    died: 1809,
+    father: "Jan van Brussel",
+    mother: "Elisabeth Haverbeke",
+  },
+];
+
```

Comment:
These data are included into separate file in your folder, no need to paste them here. You can use `require()` statement to load it as module. (Usually used by SSJS/NodeJS, but compatible with ES5  and will work for you as well). Or you can link 2 js files to each other via html document, using them as resource and have a main function call in the document body.